js/packages/teams-ai/src/types/ClientCitation.ts

```diff
@@ -44,22 +68,30 @@ export interface ClientCitation {
         url?: string;
 
         /**
-         * Content of the citation. Must be clipped if longer than 480 characters.
+         * Content of the citation. (max length 160)
```

Comment:
Let's call this either abstract or excerpt of the citation as it's not the full content. Also, remove the statement about 480 characters (it's an internal SMBA implementation detail) and only say max length is 160.

Reply:
the red is the line that was removed, the green is what was added